tests/suites/test_suite_pkcs11_client.function

```diff
@@ -336,106 +336,34 @@ void pk_signX_verifyX( int key_type )
     CK_OBJECT_HANDLE hPublicKey = CK_INVALID_HANDLE;
     CK_OBJECT_HANDLE hPrivateKey = CK_INVALID_HANDLE;
     unsigned char hash_value[32] = "Fake hash, it doesn't matter....";
-    unsigned char sig_buffer[RSA_KEY_SIZE_BYTES];
+    unsigned char sig_buffer[4096];
```

Comment:
Please don't introduce new magic numbers. You can probably do something based on `MBEDTLS_MPI_MAX_BYTES` (which is what effectively sets the limit of the size of RSA keys in mbed TLS).